### PR TITLE
EZP-27975: Support post_max_size value set to 0 during content/publish in UI

### PR DESCRIPTION
**Pull request by andrerom**
Correct logic of c95ee808ea836a2f3d3b83876bafbfb208dde4a0

### CODE REVIEW

File: kernel/content/attribute_edit.php [hunk 1]
```diff
@@ -221,7 +221,7 @@
     $postMaxSize = trim( ini_get( 'post_max_size' ) );
 
     // As of PHP 5.3.2 it is possible to configure unlimited post_max_size, further checking
-    if ($postMaxSize == 0)
+    if ($postMaxSize != 0)
     {
         $postMaxSizeBytes = $postMaxSize;
```
Comment (blankse) [inline kernel/content/attribute_edit.php]: This could help for the failing test:
`$postMaxSizeBytes = (int)$postMaxSize;`

Comment (andrerom) [reply kernel/content/attribute_edit.php]: 👍  , but I'll first var dump what the hell is configured here. maybe it's 0b or something, but have to know 😆

Comment (blankse) [reply kernel/content/attribute_edit.php]: It fails in line 239. So it is something with 'm' at the end :)

**Comment by andrerom** [comment]
thanks @blankse

**Pull request merged**